src/components/hv-text-field/index.tsx

```diff
@@ -77,6 +77,14 @@ const HvTextField = (props: HvComponentProps) => {
     null as TextInput | null,
   );
 
+  const prevDefaultValue = useRef<string | undefined>(defaultValue);
```

Comment:
Can this be a state instead of a ref?

Reply:
It could, but I'm not sure why this would be better?

Reply:
Thinking more about it I think ref makes sense here since we don't want to cause a re-render on updating the value, nvm.